### PR TITLE
Update to 0.4.0

### PR DESCRIPTION
**Pull request by jtpio**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jtpio** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -43,3 +42,4 @@ extra:
   recipe-maintainers:
     - mariobuikhuizen
     - jtpio
+    - fcollonval
```
Comment (jtpio) [inline recipe/meta.yaml:45]: cc @fcollonval let me know if it's ok to add you to the list of maintainers

Comment (fcollonval) [reply recipe/meta.yaml:45]: Ok for me thanks @jtpio

Comment (jtpio) [reply recipe/meta.yaml:45]: Nice, thanks!

**Pull request merged**